### PR TITLE
Doppler page

### LINKED ISSUE #21: Doppler Page
We need a page describing:

- What Doppler is
- How to install it
  - Include creating an account
- How to use it

The page already exists. It just needs the information filled in.

### PR DESCRIPTION
**Pull request by SebastianNeiswanger**
closes #21 
Includes instructions on:
- What it is
  - Includes analogy that Westin helped create
- Installation
- How to Use

### CODE REVIEW

File: Dev-Tools/doppler/tool.md [hunk 1]
```diff
@@ -4,12 +4,105 @@
...
+    K(Kid - CSG developer)
+    D[(Dad - other software)]
+    M -- cat w/ letter --> K <--> D
+    M -.-x O
+```
+
 ## Installation
 
-### Desktop
+### Windows
```
Comment (SpBills) [inline Dev-Tools/doppler/tool.md:43]: I disagree with using these instructions. I think we should use the default Doppler install path.

**Review by SpBills — CHANGES_REQUESTED** [review]

File: Dev-Tools/doppler/tool.md (hunk 1 above)
Comment (SebastianNeiswanger) [reply Dev-Tools/doppler/tool.md:43]: How did you install it? I had to use scoop.

Comment (SebastianNeiswanger) [reply Dev-Tools/doppler/tool.md:43]: I changed the install method to use winget

File: Dev-Tools/doppler/tool.md [hunk 2]
```diff
@@ -4,12 +4,108 @@
...
+1. Open a terminal and navigate to the directory containing the project. You may do this by opening a terminal within VSCode.
+2. Run `doppler setup`
+3. Select the project that you are working in, and then press enter
+4. Select the `dev` config, and press enter
+
+### Running commands
+
+The default command for anything now will have `doppler run &&` attached to the front of it. Doppler needs to give you the secrets before you can do things like run the application or access prisma. Here a few examples of popular commands
+
+- `doppler run && prisma studio`
```
Comment (ashlyndevries) [inline Dev-Tools/doppler/tool.md]: @SpBills said these are supposed to be dashes, ie. `--` rather than `&&`?

Comment (SebastianNeiswanger) [reply Dev-Tools/doppler/tool.md]: Oops

Comment (SebastianNeiswanger) [reply Dev-Tools/doppler/tool.md]: Fixed it.

**Review by ashlyndevries — CHANGES_REQUESTED** [review]
FAQ section: 

- Doppler said it was successfully installed, why does `doppler --version` not work? Need to restart terminal.

doppler2

- I typed `winget install doppler` and was told I need to refine my input? Type `winget installer Doppler.doppler`.

doppler1

**Comment by SebastianNeiswanger** [comment]
I will make sure to update these and inform you of when it is ready.

**Comment by SebastianNeiswanger** [comment]
@ashlyndevries 
Updated the instructions rather than adding them to the FAQ.

**Review by SpBills — APPROVED** [review]
i likey

File: Dev-Tools/doppler/tool.md [hunk 3]
```diff
@@ -100,8 +100,8 @@ When logging in you will want to use your Cedarville account. You should have re
 
 The default command for anything now will have `doppler run &&` attached to the front of it. Doppler needs to give you the secrets before you can do things like run the application or access prisma. Here a few examples of popular commands
```
Comment (ashlyndevries) [inline Dev-Tools/doppler/tool.md]: Needs to be `doppler run --`

**Review by ashlyndevries — CHANGES_REQUESTED** [review]
One little change:)

**Review by ashlyndevries — APPROVED** [review]
LGTM.

**Pull request merged**